grid/social-media/followers/profile.supr

```diff
@@ -64,6 +90,29 @@ usecase GetFollowers safe {
     A human-readable explanation specific to this occurrence of the problem.
     """
     detail
+
+    """
+    Code
+    Error code.
+    """
+    code! ErrorCode!
+
+    """
+    Rate limit
+    Rate limit error details.
+    """
+    rateLimit {
```

Comment:
Shouldn't this share the same model with `rateLimit` in result?

Reply:
Good point. `RateLimit` model is shared in error and result outcomes.